cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -37,8 +37,8 @@
 
 #include "cob_obstacle_distance/Registration.h"
 
-#define DEBUG_BASE_ACTIVE    0
-#define DEBUG_BASE_COMP     0
+
+//ToDo: Should we re-add DEBUG_BASE_COMPENSATION stuff?
```

Comment:
Why asking questions in code? What shall the reader do with this question? Code is no discussion platform!